scripts/ci/run_csi_test_client.sh

```diff
@@ -8,5 +8,5 @@ else
     csi_params='csi_params'
```

Comment:
I think it would be better to pass the full tests to skip file path, from this file to the dockerfile

Reply:
that way you could easily add logic to unify a new general "tests to skip" file with a storage-specific "tests to skip" file

Reply:
done

Reply:
could you also split them so that we won't maintain duplications?

Reply:
what do you mean?

Reply:
I mean that the common tests to skip should be maintained in a separate file, and not duplicated for each array type

Reply:
now I understand your first comments, sorry.
done.